db/fixtures/chargeback_rates.yml

```diff
@@ -198,18 +185,6 @@
           :finish: Infinity
           :fixed_rate: 1.0
           :variable_rate: 0.02
-    - :description: Allocated CPU Count
-      :group: cpu
-      :source: allocated
-      :metric: derived_vm_numvcpus
-      :per_time: hourly
-      :per_unit: cpu
-      :type_currency: Dollars
-      :tiers:
-        - :start: 0
-          :finish: Infinity
-          :fixed_rate: 1.0
-          :variable_rate: 0.0
```

Comment:
Its a little different for containers - metrics for them dont act like metrics for vms. yes, we collect `derived_vm_numvcpus` but its incorrect to charge for this field since the container shares the assigned cores with the rest of the pod.
`cpu_usage_rate_average` is the actual stat from the container so `v_derived_cpu_total_cores_used = cpu_usage_rate_average * derived_vm_numvcpus` is correct.

@simon3z Keep me honest on this explanation